django/santropolFeast/page/templates/registration/login.html

```diff
@@ -36,6 +36,41 @@ <h2 class="ui teal image header">
         <button class="ui fluid large teal button" type="submit">Login</button>
     </div>
 </form>
+
```

Comment:
Unnecessary, as semanticUi JS and Jquery are already loaded by the base template.
But be careful to execute the JS on document.ready.